tessellation/src/basic_shapes.rs

```diff
@@ -771,7 +742,7 @@ pub fn fill_circle<Output: GeometryBuilder<FillVertex>>(
         (PI * 0.5, PI),
     ];
 
-    let arc_len = 0.5 * PI * radius;
+    let arc_len = 2.0 * PI * radius;
```

Comment:
Same there: the arc length is for a quarter of the circle and we do each quarter separately, so `0.5 * PI * radius` should be good.